runner/runners/local_output_test.go

```diff
@@ -0,0 +1,60 @@
+package runners
+
+import (
+	"os"
+	"testing"
+
+	"github.com/twitter/scoot/common/log/hooks"
+	"github.com/twitter/scoot/os/temp"
+
+	log "github.com/sirupsen/logrus"
+)
+
+func init() {
+	log.AddHook(hooks.NewContextHook())
+	logrusLevel, _ := log.ParseLevel("debug")
+	log.SetLevel(logrusLevel)
+}
+
+func TestLocalOutputCreator(t *testing.T) {
+	td, err := temp.TempDirDefault()
+	if err != nil {
+		t.Fatalf("Unable to create temp dir: %v", err)
+	}
```

Comment:
+ defer remove all td.Dir for both tests?

Reply:
yup - thanks! meant to put that in and blanked